samples/quickstart.js

```diff
@@ -31,13 +31,14 @@ async function main(
    * TODO(developer): Uncomment these variables before running the sample.
    */
   // const projectId = 'YOUR_PROJECT_ID';
-  // const location = 'YOUR_PROJECT_LOCATION';
+  // const location = 'Project location--Must be in format \'us\' or \'eu\'';
   // const gcsInputUri = 'YOUR_SOURCE_PDF';
 
   const {
     DocumentUnderstandingServiceClient,
   } = require('@google-cloud/documentai');
-  const client = new DocumentUnderstandingServiceClient();
+  const clientOptions = {apiEndpoint: 'us-documentai.googleapis.com'};
```

Comment:
If ind this weird that we're pinning to `us`, I think instead we should do this:

```
{apiEndpiont: `${location}-documentai.googleapis.com`}
```

I'm a little concerned though, because this seems pretty non-standard ... perhaps we launch like this, but we should definitely raise this anomalous API... it tends to be a `parent` path which describes the location/project/etc.

Reply:
This is going to become the new standard.

Reply:
Example of this pattern:
https://cloud.google.com/natural-language/automl/docs/locations